protocol.md

```diff
@@ -103,6 +103,31 @@ To verify:
 Either standard or URL-safe base64 encodings are allowed. Signers may use
 either, and verifiers **MUST** accept either.
 
+## Multi-signature Verification
+
+Multi-signature enhances the security by allowing multiple signers to sign the
+same payload. The resulting signatures are encoded and transmitted, preferably
+using the recommended [JSON envelope](envelope.md).
+
+A `(t, n)`-ENVELOPE is valid if the enclosed signatures pass the verification
+against at least `t` of `n` unique trusted public keys.
+
+To verify a `(t, n)`-ENVELOPE:
```

Comment:
Where does `t` come on? Application-specific logic?

Reply:
`t` is from the application-specific logic.

Reply:
Got it. Would you please document this? Thanks.

Reply:
Updated by d2a3392